middlewares/accesslog/logger.go

```diff
@@ -222,7 +234,11 @@ func (l *LogHandler) logTheRoundTrip(logDataTable *LogData, crr *captureRequestR
 	}
 
 	for k := range logDataTable.Request {
-		fields["request_"+k] = logDataTable.Request.Get(k)
+		if _, shouldredact := l.redact[k]; shouldredact {
+			fields["request_"+k] = "REDACTED_BY_TRAEFIK"
+		} else {
+			fields["request_"+k] = logDataTable.Request.Get(k)
+		}
```

Comment:
We could also implement this block as following:

1. Store the redacted message in a variable.
2. If we should not redact: overwrite the variable with the value from the table.
3. Finally, set the value on the field.

This approach would enable us to mention the fields key once only.

WDYT?